### PR TITLE
Add a locale switcher

### PR DESCRIPTION
**Pull request by mitchelkuijpers**
* Listens to the app state and updates the selector if the locale changes
* If you change the locale via the switcher it will run a transaction on the application reconciler with `i18n/change-locale`

NOTE: I am not quite sure how to do the code formatting, maybe it would be a idea to have cljfmt configuration or something like that.

Resolves: #37 

fulcro-inspect-i18n-switcher

### CODE REVIEW

File: devcards/fulcro/inspect/card_helpers.cljc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+(ns fulcro.inspect.card-helpers
+  (:require [fulcro-css.css :as css]
+            #?(:clj [fulcro.client.dom-server :as dom]
+               :cljs [fulcro.client.dom :as dom])
+            [fulcro.i18n :as i18n]
+            [fulcro.client.primitives :as fp])
+  #?(:cljs (:require-macros [fulcro.inspect.card-helpers :refer [load-locale]])))
```
Comment (wilkerlucio) [inline devcards/fulcro/inspect/card_helpers.cljc]: refer is not necessary, just require the namespace will work

File: devcards/fulcro/inspect/ui/i18n_cards.cljs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+(ns fulcro.inspect.ui.i18n-cards
+  (:require
+   [devcards.core :refer-macros [defcard]]
+   [fulcro-css.css :as css]
+   [fulcro.client.cards :refer-macros [defcard-fulcro]]
+   [fulcro.client.primitives :as fp :refer [defsc]]
+   [fulcro.client.data-fetch :as fetch]
+   [fulcro.client.mutations :as mutations]
```
Comment (wilkerlucio) [inline devcards/fulcro/inspect/ui/i18n_cards.cljs]: can we a little cleanup on these requires? 💅

File: devcards/fulcro/inspect/card_helpers.cljc (hunk 1 above)
Comment (mitchelkuijpers) [reply devcards/fulcro/inspect/card_helpers.cljc]: Did not know this :+1: thnx

Comment (mitchelkuijpers) [reply devcards/fulcro/inspect/card_helpers.cljc]: But now I am getting:

```
WARNING: Can't take value of macro fulcro.inspect.card-helpers/load-locale at line 20 /home/mitchel/Development/fulcro-inspect/devcards/fulcro/inspect/card_helpers.cljc
WARNING: Can't take value of macro fulcro.inspect.card-helpers/load-locale at line 21 /home/mitchel/Development/fulcro-inspect/devcards/fulcro/inspect/card_helpers.cljc
WARNING: Can't take value of macro fulcro.inspect.card-helpers/load-locale at line 22 /home/mitchel/Development/fulcro-inspect/devcards/fulcro/inspect/card_helpers.cljc
WARNING: Can't take value of macro fulcro.inspect.card-helpers/load-locale at line 23 /home/mitchel/Development/fulcro-inspect/devcards/fulcro/inspect/card_helpers.cljc
```

**Comment by mitchelkuijpers** [comment]
I'll play around with cljfmt if I can get it to do minimal changes to the code, Also I have not remove refer for the require macros, because then I had to add the complete namespace where I called the macro. If you want I can still change that.

**Comment by mitchelkuijpers** [comment]
Would you mind going ahead and just format this to your liking? Then I can make a separate PR with formatting

**Comment by wilkerlucio** [comment]
Thanks!

**Pull request merged**